### PR TITLE
Remove REEEEEEEEEEEEEEEEEEEEEEEEE

### PR DESCRIPTION
**Pull request by Armyguy**
Just remove it

## About The Pull Request

I removed the frogs reeeeeeeeeee

## Why It's Good For The Game

It removes the frogs reeeeeeeeeee

## Changelog
:cl:
Removed the frogs reeeeeeeeeee
/:cl:

### CODE REVIEW

**Comment by Armyguy** [comment]
Ok Im going to need to remove the mob from the maps its on.
THAT i will do later so for now DNM.

**Comment by KasparoVy** [comment]
:-1::frog:

**Comment by atakiya** [comment]
You could've just commented out the line that plays the annoying sound, _**if**_ that is the reason for the removal, without invoking a ton of other issues.

PS: GitHub has a "draft PR" feature if your PR is not ready to be merged.
Top right:
image

**Comment by Armyguy** [comment]
> You could've just commented out the line that plays the annoying sound, _**if**_ that is the reason for the removal, without invoking a ton of other issues.
> 
> PS: GitHub has a "draft PR" feature if your PR is not ready to be merged.
> Top right:
> image

Yea but i forgot that its just ON a map

**Comment by Armyguy** [comment]
There we go sound removed

**Comment by Armyguy** [comment]
There we go, it just removes the sound now making it 1000% better.

**Comment by NotRanged** [comment]
Bad PR, keep frog

File: code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm [hunk 1]
```diff
@@ -24,12 +24,12 @@
 	density = FALSE
 	ventcrawler = VENTCRAWLER_ALWAYS
 	faction = list("hostile")
-	attack_sound = 'sound/effects/reee.ogg'
+	//attack_sound = 'sound/effects/reee.ogg'
```
Comment (atakiya) [inline code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm]:
```suggestion
	//attack_sound = 'sound/effects/reee.ogg' //SKYRAT EDIT REMOVAL 
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm [hunk 2]
```diff
@@ -24,12 +24,12 @@
...
 	butcher_results = list(/obj/item/food/nugget = 1)
 	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
 	mob_size = MOB_SIZE_TINY
 	gold_core_spawnable = FRIENDLY_SPAWN
-	var/stepped_sound = 'sound/effects/huuu.ogg'
+	//var/stepped_sound = 'sound/effects/huuu.ogg'
```
Comment (atakiya) [inline code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm]:
```suggestion
	//var/stepped_sound = 'sound/effects/huuu.ogg' //SKYRAT EDIT REMOVAL 
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm [hunk 3]
```diff
@@ -40,10 +40,11 @@
 		icon_living = "rare_frog"
 		icon_dead = "rare_frog_dead"
 		butcher_results = list(/obj/item/food/nugget = 5)
-
+/*
```
Comment (atakiya) [inline code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm:45]:
```suggestion

//SKYRAT EDIT REMOVAL BEGIN
/*
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm [hunk 4]
```diff
@@ -40,10 +40,11 @@
...
 /mob/living/simple_animal/hostile/retaliate/frog/Crossed(AM as mob|obj)
 	. = ..()
 	if(!stat && isliving(AM))
 		var/mob/living/L = AM
 		if(L.mob_size > MOB_SIZE_TINY)
 			playsound(src, stepped_sound, 50, TRUE)
+*/
```
Comment (atakiya) [inline code/modules/mob/living/simple_animal/hostile/retaliate/frog.dm:52]:
```suggestion
*/
//SKYRAT EDIT REMOVAL END
```

**Review by atakiya — CHANGES_REQUESTED** [review]
@Armyguy By the way we have a [modularisation guide](https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/readme.md).
See [this part on how to add/remove/change non-modular code](https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/readme.md#non-modular-changes-to-the-core-code---important).

This time I've gone ahead and commented the appropiate additions to make it conform, so if you're fine with the changes you did, just apply them! Aside of that this looks fine!

**Comment by atakiya** [comment]
@Armyguy The first two var removals also have to be marked with the skyrat comment, I've updated my suggestion to be more streamlined.

**Review by atakiya — APPROVED** [review]

**Pull request merged**